wagtail/admin/ui/sidebar.py

```diff
@@ -85,6 +85,47 @@ def __eq__(self, other):
         )
 
 
+@adapter("wagtail.sidebar.ActionMenuItem", base=BaseSidebarAdapter)
+class ActionMenuItem(MenuItem):
```

Comment:
I haven't added a corresponding class in `wagtail/admin/menu.py`. This telepath component is generic enough to be reusable, it's basically an empty form with a submit button and a customisable `action` and `method`.

Not sure if it's something we'd like to make available now, though.